gridy/Sources/Presentations/PlanBoard/TimelineLayout/RightToolBarArea/ScrollDatePickerView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ScrollDatePickerView.swift
+//  gridy
+//
+//  Created by SY AN on 10/7/23.
+//
+
+import SwiftUI
+
+struct ScrollDatePickerView: View {
+    @State private var selectedDate = Date()
+    @Binding var proxy: ScrollViewProxy?
+    var body: some View {
+        VStack {
+            DatePicker(
+                "",
+                selection: $selectedDate,
+                displayedComponents: [.date]
+            )
+            
+            Button(action: {
+                withAnimation {
+                    let scrollIndex = selectedDate.integerDate
+                    proxy?.scrollTo(scrollIndex, anchor: .leading)
+                }
+            }, label: {
+                Text("이동")
+            })
```

Comment:
버튼의 action이 외부에서 정의된 클로저가 아니라면 [trailing closure](https://www.hackingwithswift.com/sixty/6/5/trailing-closure-syntax)로 작성해요!
괄호 사용을 최소화하면 코드 읽기가 수월해집니다
```swift
Button {
  // action
} label : {
  // label 
}
```

Reply:
```suggestion
            Button(action: buttonAction) {
                Text("이동")
            })
            
            // body 밖에서
            private func buttonAction() {
                    withAnimation {
                            let scrollIndex = selectedDate.integerDate
                            proxy?.scrollTo(scrollIndex, anchor: .leading)
                    }
            }
```

버튼이라서 action에 대한 클로저를 분리해서 사용하는 것도 좋겠습니다
버튼에 대한 동작이 많아지면 함수에서만 수정하면 돼서 유지보수에도 좋을 것 같고,
label에 대해서는 trailing closure로 작성할 수도 있어 가독성도 좋아지네요

Reply:
완료 🥳